Source/WebCore/dom/Document.cpp

```diff
@@ -923,16 +923,18 @@ void Document::parseMarkupUnsafe(const String& markup, OptionSet<ParserContentPo
             html->appendChild(head);
             html->appendChild(body);
             appendChild(html);
+            return;
```

Comment:
Question for reviewer: because of this I can nest `usedFastPath` above inside the first conditional. Should I just not have a variable at all and put `tryFastParsingHTMLFragment` directly inside `if (LIKELY`?